src/internal/observable/bindNodeCallback.ts

```diff
@@ -255,98 +252,10 @@ export function bindNodeCallback(callbackFunc: Function, scheduler?: SchedulerLi
  * deliver.
  * @name bindNodeCallback
  */
-export function bindNodeCallback<T>(
+export function bindNodeCallback(
   callbackFunc: Function,
```

Comment:
issue: Not related to the refactor, but can we get rid of the `Function` usage while we're here? It's a completely useless type. IIRC, it's something like `{ name: string }`. (I think I'll write a lint rule, now, to ban it.) `(...args: any[]) => any` would be better. Maybe we should have an internal type for an arbitrary function?

I can see that the `bindCallback` and `bindNodeCallback` implementation also make liberal use of `any` instead of an arbitrary function type. They should be fixed up, too, IMO. Maybe in another commit.

Reply:
Sure. I think it's just there from "way back when" TBH. I was pretty new to TypeScript whenever that was written.

Reply:
I'll leave it for another PR though.